components/authz-service/cmd/authz-service/main.go

```diff
@@ -115,7 +116,7 @@ Please pass a config file as the only argument to this command.`))
 
 	// if server.GRPC() returns, it's with an error
 	fail(server.GRPC(ctx, cfg.GRPC, l, connFactory, engine, migrationConfig,
-		dataMigrationConfig, cfg.EventAddress))
+		dataMigrationConfig, cfg.EventAddress, cfg.ConfigFile))
```

Comment:
Is it worth being more specific about what kind of `ConfigFile` this is? The many config things in here is a bit confusing

Reply:
I was thinking this would be a general config file for the service. For example, the ingest-service is using the config file for storing job scheduler information and for the project updater state. The config manager that uses this file is threadsafe to allow all the different parts of the service to use the same one.